Libraries/Image/Image.ios.js

```diff
@@ -118,6 +118,20 @@ var Image = React.createClass({
      * 'stretch': Scale width and height independently, This may change the
      * aspect ratio of the src.
      */
+    getSize: PropTypes.func,
+    /**
```

Comment:
Also, if `getSize` isn't a prop we shouldn't pretend it's one. Better to have no docs than wrong docs...